### PR TITLE
Please help: Add Review form not working

### PR DESCRIPTION
**Pull request by adaau**
@dshcheung Please take a look at the code and see if you can find what is wrong. 

Thanks!

### CODE REVIEW

File: app/assets/stylesheets/reviews.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    border-top-style: solid;
+    border-width: 1px;
+    border-color: #D8D8D8;
+  }
+
+  li.comment {
+  border-bottom-style: solid;
+  border-width: 1px;
+  border-color: #D8D8D8;
+  padding-top: 10px;
```
Comment (dshcheung) [inline app/assets/stylesheets/reviews.scss]: indentations here

File: app/views/reviews/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <div class="col-xs-12 col-sm-10 col-md-8 features">
+
+    <%= render 'form' %>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-xs-12">
+      <ul class="list-unstyled reviews">
+        <% @playspace.reviews.reverse_each do |review| %>
```
Comment (dshcheung) [inline app/views/reviews/index.html.erb]: if you have `@reviews = @playspace.reviews` in your `reviews_controller.rb`, then please use that.

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  #index
+  def index
+    @reviews = @playspace.reviews
+    @review  = @playspace.reviews.new
+  end
+
+  #new
+  def end
+    @review = @playspace.reviews.new
+  end
```
Comment (dshcheung) [inline app/controllers/reviews_controller.rb]: the method name should be `new` instead of `end`

File: app/assets/stylesheets/reviews.scss (hunk 1 above)
Comment (adaau) [inline app/assets/stylesheets/reviews.scss]: oh #$&@ I am so *not* doing 60 push-ups.

File: app/controllers/reviews_controller.rb (hunk 3 above)
Comment (adaau) [inline app/controllers/reviews_controller.rb]: omg...

File: app/views/reviews/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<%= bootstrap_form_for @review, url: playspace_reviews_path(@playspace.id) do |f| %>
+  <div>
+    <%= f.text_field :comment %>
+  </div>
+  <div>
```
Comment (dshcheung) [inline app/views/reviews/_form.html.erb]: if you use `bootstrap_form_for` you don't need to have a `<div></div>` around your `f.text_field`

File: app/views/reviews/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      <%= link_to 'Back', playspace_path(@playspace.id), class: "btn btn-default" %>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-xs-12 col-sm-10 col-md-8 features">
+      <h6>Features</h6>
+      <span class="features-list"><%= features_list(@playspace) %></span>
+    </div>
+  </div>
```
Comment (dshcheung) [inline app/views/reviews/index.html.erb]: i don't think you need features inside your reviews-index, features should only be in playspace-index

**Comment by f3r** [comment]
@adaau please make sure you rebase this branch and fix the comments, ping me when done pls

**Pull request closed without merging**